README.md

```diff
@@ -37,3 +37,7 @@ npm run test:cy:run
 ```
 
 Both of the above assume that the development server is running, so be sure to start that first. Alternatively, `npm run test:e2e:ci` will start the dev server, run the test suite, and shut the server down afterward all in one command.
+
+## Linting & Styling
```

Comment:
This is great @heymatthenry! 

Suggestion: Let's include a snippet on our use of pa11y-ci.

it's going to be super helpful when we start to compile compliance documentation

Reply:
@Jkrzy Good thought. Added!